src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1117,8 +1117,20 @@ private void AddPatchNode(Patch patch, Vector2 position)
         {
             var playerPopulation = patch.GetSpeciesSimulationPopulation(playerSpecies);
 
+            var speciesSizeModifier = 1.0;
+
+            if (playerSpecies is MicrobeSpecies microbeSpecies)
+            {
+                speciesSizeModifier = Math.Sqrt(microbeSpecies.BaseHexSize);
```

Comment:
Base hex size is a very expensive thing to calculate, so I'd like to ask that you check the player population first and only do the hex size calculation when it would actually result in any dots on the map.